version: '3.1'

services:
  proxy:
    image: traefik:1.5.4-alpine

    container_name: traefik

    command: --docker --logLevel=ERROR
    
    networks:
      - se_front
      - se_back

    ports:
      - "80:80"
      - "8080:8080"

    labels:
      - "traefik.enable=false"
      
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - ./config/traefik.toml:/traefik.toml

networks:
  se_front:
    external: true
  se_back:
    external: true
